src/package.xml

```diff
@@ -318,6 +318,7 @@
         <members>RelationshipsLookupDescription</members>
         <members>addrHhAdmAccountOnly</members>
         <members>addrNotEnabled</members>
+        <members>addrNotEnabledContact</members>
```

Comment:
I believe this is the only line that should be committed for package.xml. The extra lines are not needed.